### ISSUE TITLE
Tari Base Node doesn't run on Ubuntu 18.04

### ISSUE THREAD

**Issue by stringhandler** [tari-project/tari#3302]
logged via telegram group:

```
Samuel Harrison, [06.09.21 09:17]
Tari base node doesnt run on ubuntu 18.04 pc's

Samuel Harrison, [06.09.21 09:17]
It requires the latest glibc drivers of ubuntu 20.4 and above

Samuel Harrison, [06.09.21 09:19]
Tari devs, please remember backward compatibility when compiling
```

When running the binary

**Comment by delta1** [comment]
Likely since we are building on 20.04, will attempt to change the build process to use 18.04

**Comment by delta1** [comment]
error building croaring on 18.04 due to old clang version (6) - putting in a PR to change to `clang-10`

**Comment by delta1** [comment]
Should be resolved by #3305 will test
